src/Snapshot.php

```diff
@@ -73,6 +79,11 @@ protected function loadAsync(?string $connectionName = null): void
             $dbDumpContents = gzdecode($dbDumpContents);
         }
 
+        if (empty(trim($dbDumpContents))) {
+            // Ignoriere leeren Snapshot
```

Comment:
```suggestion
```

This comment is a bit too German for my liking. 👀 Please either change it to English or remove it.